assembl/static2/js/app/components/search.jsx

```diff
@@ -130,27 +131,54 @@ const ImageType = (props) => {
   return <img className={props.className} src={`/static2/img/icon-${props.type}.svg`} alt="" />;
 };
 
+const getFieldAnyLang = (source, prop, locale) => {
+  let result;
+  if (!source) {
+    // Why does this happen?
+    return result;
+  }
+  if (source[`${prop}_${locale}`]) {
```

Comment:
Template strings aren't too performant yet. I would make this a variable and use accordingly.
https://stackoverflow.com/a/29083467

Reply:
Yes, but your eslint settings impose templates. shrug.

Reply:
Please keep the template litteral. Performance in browser can change from version to version, and you're arguing for some microseconds here... If this code was in a requestFrameAnimation loop, for VR for example, maybe, and still... And please don't make optimizations based on some stackoverflow comment that can be outdated. Before optimizing something, measure measure measure! ;-) And by the way, the browser is really executing a concatenation here, because babel transform this in ES5. :-D